changelog.md

```diff
@@ -1,7 +1,25 @@
 # 0.13.2
 
+## Major Changes
+* Drop support for Python 3.6 (#726)
+* Added Colab to try SMAC in your browser! (#697)
+* `BOHB4HPO` facade has been renamed to `SMAC4MF` facade
+
 ## Minor Changes
-* Require emcee >= 3.0.0
+* Require `emcee` >= 3.0.0 (#723)
+* Require `scipy` >= 1.7 (#729)
+* Added gradient boosting example
+* `lazy_import` dependency dropped (#741)
+* Dropped `pyDOE` requirement, using `scipy` instead (#735)
```

Comment:
Maybe better to specify that this update is for LHD design?

Reply:
Sorry I'm unfamiliar with the internals of SMAC, I went with

* Replaced `pyDOE` requirement with `scipy` for LHD design (#735)